### ISSUE TITLE
How to configure "API Routes Response Size"

### ISSUE THREAD

**Issue by ndaidong** [umami-software/umami#2747]
### Describe the Bug

Sometimes I could not view realtime page.
It looks like the data is too large, exceeds the limit of Nextjs router, as they explain in this guide:

https://nextjs.org/docs/messages/api-routes-response-size-limit

How to change that config - `responseLimit` - in Umami?

### Database

PostgreSQL

### Relevant log output

```shell
API response for /api/realtime/58a20b98-****-****-****-e35d69ea6bab?startAt=0 exceeds 4MB. API Routes are meant to respond quickly. https://nextjs.org/docs/messages/api-routes-response-size-limit
```

### How are you deploying your application? (if relevant)

Docker

**Comment by ndaidong** [comment]
@mikecao yes, that often occurs when there are more than 1000 visitors on the website.

**Comment by mikecao** [comment]
Fixed in v2.12.0.
